src/completers/completeReducer/README.md

```diff
@@ -1,56 +1,51 @@
+const actions = createTypes(completeTypes({
+  primaryActions: ['PRIMARY_ACTION'],
+  pollingActions: ['POLLING_ACTION'],
+  modalActions: ['MODAL_ACTION'],
+  ignoredActions: ['IGNORED_ACTION'],
+  customCompleters: [{actions: ['CUSTOM_ACTION'], completer: action => `${action}_COMPLETED`}]
+}), '@@NAMESPACE');
 const reducerDescription = {
-  primaryActions: [actions.FETCH],
+  primaryActions: [actions.PRIMARY_ACTION],
+  pollingActions: [actions.POLLING_ACTION],
+  modalActions: [actions.MODAL_ACTION],
   override: {
-    [actions.OTHER]: (state, action) => ({ ...state, count: action.payload })
+    [actions.IGNORED_ACTION]: (state, action) => ({ ...state, someTarget: action.payload }),
+    [actions.CUSTOM_ACTION_COMPLETED]: (state, action) => ({ ...state, someOtherTarget: action.payload }),
+    [actions.PRIMARY_ACTION_FAILURE]: (state, action) => ({ ...state, someOtherTarget: action.payload }), // overrides the default onFailure() of PRIMARY_ACTION_FAILURE
   }
 }
 
-const reducer = createReducer(initialState, completeReducer(reducerDescription));
-```
-
-reducer handlers will be equivalent to:  
-```js
-  {
-    [actions.FETCH]: onLoading(),
-    [actions.FETCH_SUCCESS]: onSuccess(),
-    [actions.FETCH_FAILURE]: onFailure(),
-    [actions.OTHER]: (state, action) => ({ ...state, count: action.payload })
-  }
-```  
-And the reducer created will behave like:  
-```js
-const reducer(state = initialState, action) {
-  switch (action.type) {
-    case actions.FETCH:
-      return { ...state, [`${action.target}Loading`]: true };
-    case actions.FETCH_SUCCESS:
-      return {
-        ...state,
-        [`${action.target}Loading`]: false,
-        [`${action.target}Error`]: null,
-        [`${action.target}`]: action.payload,
-      };
-    case actions.FETCH_FAILURE:
-      return {
-        ...state,
-        [`${action.target}Loading`]: false,
-        [`${action.target}Error`]: action.payload
-      };
-    case actions.OTHER:
-      return { ...state, count: action.payload };
-    default:
-      return state;
+const completedReducer = completeReducer(reducerDescription);
+/*
+{
+  [actions.PRIMARY_ACTION]: onLoading(),
+  [actions.PRIMARY_ACTION_SUCCESS]: onSuccess(),
+  [actions.PRIMARY_ACTION_FAILURE]: onFailure(), //this is overwritten afterwards
+  [actions.MODAL_ACTION_OPEN]: onSuscribe(),
+  [actions.MODAL_ACTION_CLOSE]: onUnsubscribes(),
+  [actions.POLLING_ACTION]: onLoading(),
+  [actions.POLLING_ACTION_SUCCESS]: onSuccess(),
+  [actions.POLLING_ACTION_FAILURE]: onFailure(),
+  [actions.POLLING_ACTION_RETRY]: onRetry(),
+  [actions.POLLING_ACTION_CANCEL]: onCancel(),
+  [actions.IGNORED_ACTION]: (state, action) => ({ ...state, someTarget: action.payload }),
+  [actions.CUSTOM_ACTION_COMPLETED]: (state, action) => ({ ...state, someOtherTarget: action.payload }),
+  [actions.PRIMARY_ACTION_FAILURE]: (state, action) => ({ ...state, someOtherTarget: action.payload })
   }
-}
-```
+*/
```

Comment:
Is it ok all these commented code?

Reply:
It's there to show the content of `completedReducer`. I'll add an extra comment to make it clearer